README.md

```diff
@@ -76,6 +76,12 @@ Maximum number of seconds to wait for each file to upload before timing out.
 
 Default value: `30`
 
+#### `annotate-report`(boolean)
```

Comment:
I think a more appropriate name would be `annotation-link` as the current name makes me thing that it will be the report the one that will be annotated